examples/task_on_ground_continuous_random_walk.py

```diff
@@ -0,0 +1,56 @@
+"""
+Here we try different cameraY (to bring the agent to the ground), gridSize (the amount of movement),
+ continuous_movement (not just 90 degree rotations and can move diagonally) and finally a specific unity
+ build. For ours we placed many cups on the ground. Still picks random actions but shows how much
+ we can vary the environment.
+"""
+import time
+import argparse
+
+from gym_ai2thor.envs.ai2thor_env import AI2ThorEnv
+
+parser = argparse.ArgumentParser(description='Provide build path')
+parser.add_argument('--build-path', required=True,
+                    help='Required Unity build path for custom scenes (e.g. cups on ground)')
+args = parser.parse_args()
+
+if __name__ == '__main__':
+
+    config_dict = {
+        'pickup_put_interaction': True,
+        'open_close_interaction': False,  # disable opening/closing objects
+        'openable_objects': [],  # disable opening objects another way
+        'pickup_objects': [
+            "Cup"
+        ],
+        'scene_id': 'FloorPlan1',  # let's try a different room
+        'grayscale': True,
+        'resolution': [128, 128],
+        'cameraY': -0.85,
+        'gridSize': 0.1,  # 0.01
+        'continuous_movement': True,
+        'build_path': args.build_path,
+        'task': {
+            'task_name': 'PickUp',
+            'target_object': 'Cup'
```

Comment:
Pull from master. target_object is now a dictionary

Reply:
was called target_objects now. I still kinda disagree with this dict approach but whatever I'll do it